lib/reactor/builder.ex

```diff
@@ -48,9 +43,13 @@ defmodule Reactor.Builder do
 
   @doc """
   Build a new, empty Reactor.
+
+  Optionally provide an identifier for the Reactor in the form of a module name
```

Comment:
I think actually an identifier for a reactor should be anything. The goal here is to prevent recursively planning reactors, and so they might make a nested reactor where the reactor ids are like... `:a` and `:b`.

Reply:
otherwise they have to make a ref called like `a_ref` and make sure its used properly.